gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/test/js/AuthorizationServiceTests.js

```diff
@@ -123,63 +166,103 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
+                }, {
+                    listener: "jqUnit.assertDeepEq",
+                    args: ["The error is returned when the client credential record is not found in the database", gpii.tests.oauth2.authorizationService.expected.unauthorized, "{arguments}.0"],
+                    event: "{that}.events.onError"
+                }]
             }, {
                 name: "grantGpiiAppInstallationAuthorization() returns error when the client type is not \"gpiiAppInstallationClient\"",
                 sequence: [{
-                    func: "gpii.tests.oauth2.invokePromiseProducer",
-                    args: ["{authorizationService}.grantGpiiAppInstallationAuthorization", ["alice_gpii_token", "client-2"], "{that}"]
+                    func: "gpii.tests.dbOperation.invokePromiseProducer",
+                    args: ["{authorizationService}.grantGpiiAppInstallationAuthorization", ["alice_gpii_key", "gpiiAppInstallationClient-2", "clientCredential-1"], "{that}"]
                 }, {
                     listener: "jqUnit.assertDeepEq",
                     args: ["The error is returned when the client type is not \"gpiiAppInstallationClient\"", gpii.tests.oauth2.authorizationService.expected.unauthorized, "{arguments}.0"],
                     event: "{that}.events.onError"
                 }]
+            }, {
+                name: "grantGpiiAppInstallationAuthorization() returns error when the client credential type is not \"clientCredential\"",
+                sequence: [{
+                    func: "gpii.tests.dbOperation.invokePromiseProducer",
+                    args: ["{authorizationService}.grantGpiiAppInstallationAuthorization", ["alice_gpii_key", "gpiiAppInstallationClient-1", "clientCredential-3"], "{that}"]
+                }, {
+                    listener: "jqUnit.assertDeepEq",
+                    args: ["The error is returned when the client credential type is not \"clientCredential\"", gpii.tests.oauth2.authorizationService.expected.unauthorized, "{arguments}.0"],
+                    event: "{that}.events.onError"
+                }]
+            }, {
+                name: "grantGpiiAppInstallationAuthorization() returns error when the client credential does not belong to the client that requests for the authorization",
+                sequence: [{
+                    func: "gpii.tests.dbOperation.invokePromiseProducer",
+                    args: ["{authorizationService}.grantGpiiAppInstallationAuthorization", ["alice_gpii_key", "gpiiAppInstallationClient-1", "clientCredential-2"], "{that}"]
+                }, {
+                    listener: "jqUnit.assertDeepEq",
+                    args: ["The error is returned when the client credential does not belong to the client that requests for the authorization", gpii.tests.oauth2.authorizationService.expected.unauthorized, "{arguments}.0"],
```

Comment:
There are some very long lines in this file. I think it would be better to break the long `args` over multiple lines. Such as:

```
args: [
    "The error is returned when the client credential does not belong to the client that requests for the authorization",
    gpii.tests.oauth2.authorizationService.expected.unauthorized,
    "{arguments}.0"
],
```